README.md

```diff
@@ -149,4 +149,147 @@ MetrixRunParameters runParams = new MetrixRunParameters(firstVariant, variantCou
+- counter file: this file describes the list of topological actions that are allowed as preemptive actions to counter an issue. The name of this name can be chosen at metrix launch (see --help)
+- index first variant: the first variant to process in the variant file. Is chosen at launch.
+- number of variants: the number of variants to process. Is chosen at launch.
+
+*outputs:*
+- user log file: the user log file, displaying user friendly information about the finished run. These information are duplicated in the developper log file
+- developper log files: a set of more developpement-axed log files, with more information
+- result files: the result files (1 by variant processed)
+
+all input options are described in --help option
+
+##### Requirements
+
+To build metrix-simulator, you need:
+- A C++ compiler that supports C++11 ([clang](https://clang.llvm.org) 3.3 or higher, [g++](https://gcc.gnu.org) 5.0 or higher)
+- [CMake](https://cmake.org) (3.12 or higher)
+- [Make](https://www.gnu.org/software/make/)
+- [Boost](https://www.boost.org) development packages (1.66 or higher)
+
+###### Ubuntu 20.04
+```
+$> apt install -y cmake g++ git libboost-all-dev libxml2-dev make
+``` 
+
+###### Ubuntu 18.04
+```
+$> apt install -y g++ git libboost-all-dev libxml2-dev make wget
+```
+
+**Note:** Under Ubuntu 18.04, the default CMake package is too old (3.10), so you have to install it manually:
+```
+$> wget https://cmake.org/files/v3.12/cmake-3.12.0-Linux-x86_64.tar.gz
+$> tar xzf cmake-3.12.0-Linux-x86_64.tar.gz
+$> export PATH=$PWD/cmake-3.12.0-Linux-x86_64/bin:$PATH
+```
+
+###### CentOS 8
+```
+$> yum install -y boost-devel gcc-c++ git libxml2-devel make wget
+```
+
+**Note:** Under CentOS 8, the default CMake package is too old (3.11.4), so you have to install it manually:
+```
+$> wget https://cmake.org/files/v3.12/cmake-3.12.0-Linux-x86_64.tar.gz
+$> tar xzf cmake-3.12.0-Linux-x86_64.tar.gz
+$> export PATH=$PWD/cmake-3.12.0-Linux-x86_64/bin:$PATH
+```
+
+###### CentOS 7
+```
+$> yum install -y gcc-c++ git libxml2-devel make wget
+```
+**Note:** Under CentOS 7, the default `boost-devel` package is too old (1.53), so we install Boost 1.66 from `epel-release`.
+```
+$> yum install -y epel-release
+$> yum install -y boost166-devel
+$> export BOOST_INCLUDEDIR=/usr/include/boost166
+$> export BOOST_LIBRARYDIR=/usr/lib64/boost166
+```
+
+**Note:** Under CentOS 7, the default CMake package is too old (2.8.12), so you have to install it manually:
+```
+$> wget https://cmake.org/files/v3.12/cmake-3.12.0-Linux-x86_64.tar.gz
+$> tar xzf cmake-3.12.0-Linux-x86_64.tar.gz
+$> export PATH=$PWD/cmake-3.12.0-Linux-x86_64/bin:$PATH
+```
+##### Build sources
+
+1 - Clone the project
+```
+$> git clone https://github.com/powsybl/powsybl-metrix.git
+$> cd powsybl-metrix
+```
+
+2 - Configure the project
+```
+$> mkdir build
+$> cd build
+$> cmake .. -DCMAKE_INSTALL_PREFIX=<PREFIX> -DCMAKE_BUILD_TYPE=<BUILD_TYPE>
```

Comment:
If I follow the step 1 and 2, this command will fail because you don' ask to change directory into metrix-simulator

Reply:
OK